apis/accounts/auth/signup.yaml

```diff
@@ -0,0 +1,28 @@
+post:
+  tags:
+    - AccountAuth
+  operationId: "Signup"
+  summary: "Signup."
+  description: |
+    Register a new password to be used to encrypt the local wallet. The password MUST:
```

Comment:
As I mentioned in the previous comment, I think we should keep the API auth account separate from the validator key encryption.

I would say that even baking in any concept of a "wallet" is best avoided. The Lighthouse validator client doesn't pay any attention to wallets (as defined in [EIP-2386](https://eips.ethereum.org/EIPS/eip-2386)). We simply have a bunch of encrypted keystores; we don't care if they were [derived](https://eips.ethereum.org/EIPS/eip-2334) from a common seed or not.